SPECS/llvm/llvm.spec

```diff
@@ -49,7 +49,7 @@ cmake -G Ninja                              \
       -DLLVM_LINK_LLVM_DYLIB=ON             \
       -DLLVM_INCLUDE_TESTS=ON               \
       -DLLVM_BUILD_TESTS=ON                 \
-      -DLLVM_TARGETS_TO_BUILD="host"        \
+      -DLLVM_TARGETS_TO_BUILD="host;BPF"    \
```

Comment:
Should we have `all` here instead of just `host;BPF` ?

Reply:
In Mariner 1.0 and 2.0 we had: `host;AMDGPU;BPF`. I think we should stick with the same until we have a reason to change this.

Reply:
I am assuming AMDGPU and BPF were dropped to fix some ptests failures. I wasn't sure so I only added back BPF.